CMakeLists.txt

```diff
@@ -227,13 +227,18 @@ cmake_dependent_option(FORCE_LEGACY_LIBBPF "Force usage of libbpf 0.0.9 instead
 mark_as_advanced(FORCE_LEGACY_LIBBPF)
 
 cmake_dependent_option(ENABLE_NETDATA_JOURNAL_FILE_READER "Enable netdata's journal file reader implementation" False "ENABLE_PLUGIN_SYSTEMD_JOURNAL" False)
-cmake_dependent_option(ENABLE_NETDATA_JOURNAL_FILE_READER_BENCHMARKS "Enable netdata's journal file reader implementation benchmarks" False "ENABLE_NETDATA_JOURNAL_FILE_READER" False)
-mark_as_advanced(ENABLE_NETDATA_JOURNAL_FILE_READER_BENCHMARKS)
 
-if(ENABLE_PLUGIN_SYSTEMD_JOURNAL AND NOT ENABLE_NETDATA_JOURNAL_FILE_READER)
-  set(ENABLE_LIBSYSTEMD_JOURNAL_FILE_READER True)
-else()
-  set(ENABLE_LIBSYSTEMD_JOURNAL_FILE_READER False)
```

Comment:
Almost forgot, this needs to be kept because the variable is used in the `config.h` header, and at minimum is used to generate buildinfo output indicating which journal file reader is in use.

Reply:
We have `netdata -W cmakecache`.